rawtodigi_alpaka.cc

```diff
@@ -0,0 +1,527 @@
+#include <algorithm>
+#include <cstdio>
+
+#include "rawtodigi_alpaka.h"
+
+
+namespace ALPAKA_ARCHITECTURE{
```

Comment:
Is this namespace strictly speaking needed for this (toy) example? Each target architecture is compiled separately and linked into separate executables, so there name collisions between architectures should not happen.

If this is the case, could you add a comment explaining that and mention something along this being a forward-looking solution for the case that we would link e.g. CPU and GPU binaries into a single program?

Reply:
The namespace is needed also here, since in [alpakaConfig.h](https://github.com/makortel/pixel-standalone/pull/10/files#diff-8714e4c5d84b3c036853692b80a27221) there is the definition of the device's symbols for each architecture, which is required by the implementation in [main_alpaka.cc](https://github.com/makortel/pixel-standalone/pull/10/files#diff-52916791a9938d7885b5149f96d8fadc) and in [rawtodigi_alpaka.h](https://github.com/makortel/pixel-standalone/pull/10/files#diff-845843df3680d3450d13c1bbb957b715)